web/src/locales/de/data.json

```diff
@@ -367,6 +368,38 @@
     "Text": "Text",
     "Text - Tooltip": "Text - Tooltip"
   },
+  "mfa": {
+    "Each time you sign in to your Account, you'll need your password and a authentication code": "Each time you sign in to your Account, you'll need your password and a authentication code",
+    "Enable": "Enable",
```

Comment:
A lot of words can be found duplicated with existing words. Try to use existing words as much as possible